src/test/kotlin/com/vanniktech/rxriddles/Riddle102Test.kt

```diff
@@ -43,4 +44,14 @@ class Riddle102Test {
 
     o.assertResult(10 to 5)
   }
+
+  private fun <T> TestObserver<T>.awaitTerminalEvent(): Boolean {
```

Comment:
This got removed in rx3? If so, can you pull it into it's own file? (If there's another extension file in the test directory you can place it there as well)

Reply:
yup, see [notes](https://github.com/ReactiveX/RxJava/wiki/What's-different-in-3.0#test-support-methods)

Reply:
let me know if you are happy with the file name